src/Components/Stats/Stats.js

```diff
@@ -1,38 +1,353 @@
+                </div>
+                <div className={style.line} />
+                <div className={style.moodStatsSection}>
+                  <p className={style.moodWidgetLabel}>
+                    After rest interval
+                  </p>
+                  {moodStatistics.movementAverageMood2 > 0 &&
+                    <img
+                      src={determineFace(moodStatistics.movementAverageMood2).icon}
+                      className={style.moodIcon}
+                      style={{ backgroundColor: determineFace(moodStatistics.movementAverageMood2).color }}
+                    />
+                  }
+                  <p 
+                    className={style.moodStatisticValue}
+                    data-testid='movementMood2'
+                  >
+                    {moodStatistics.movementAverageMood2.toFixed(2)}
+                  </p>
+                </div>
+              </div>
+            </h4>
+            <h4 className={style.moodWidget}>
+              Meditation/Breathwork
+              <p className={style.moodAverageLabel}>
+                Average Mood
+              </p>
+              <div className={style.moodStatsContainer}>
+                <div className={style.moodStatsSection}>
+                  <p className={style.moodWidgetLabel}>
+                    Before rest interval
+                  </p>
+                  {moodStatistics.meditationAverageMood1 > 0 &&
+                    <img
+                      src={determineFace(moodStatistics.meditationAverageMood1).icon}
+                      className={style.moodIcon}
+                      style={{ backgroundColor: determineFace(moodStatistics.meditationAverageMood1).color }}
+                    />
+                  }
+                  <p 
+                    className={style.moodStatisticValue}
+                    data-testid='meditationMood1'
+                  >
+                    {moodStatistics.meditationAverageMood1.toFixed(2)}
+                  </p>
+                </div>
+                <div className={style.line} />
+                <div className={style.moodStatsSection}>
+                  <p className={style.moodWidgetLabel}>
+                    After rest interval
+                  </p>
+                  {moodStatistics.meditationAverageMood2 > 0 &&
+                    <img
+                      src={determineFace(moodStatistics.meditationAverageMood2).icon}
+                      className={style.moodIcon}
+                      style={{ backgroundColor: determineFace(moodStatistics.meditationAverageMood2).color }}
+                    />
+                  }
+                  <p 
+                    className={style.moodStatisticValue}
+                    data-testid='meditationMood2'
+                  >
+                    {moodStatistics.meditationAverageMood2.toFixed(2)}
+                  </p>
+                </div>
+              </div>
+            </h4>
+          </section>
+        </> :
```

Comment:
I really like that this rendering was done with a ternary, I've never seen it used in this way before and it's rad. However, that little `:` is rather tricky to spot, and I think it would be a little more readable/dev-empathetic to actually create two separate conditions for these different cases.